### PR TITLE
chore: Updated Gruntfile after pure-css#705.

### PR DESCRIPTION
**Pull request by medianect**
The name of the license text file was changed but not references to it in Gruntfile.js, resulting into invalid link in compiled versions headers.

### CODE REVIEW

**Comment by medianect** [comment]
An example will be better than my hard to parse sentence. Here is the header in actual released files (note the `.md` at the end of license URL):
```css
/*!
Pure v2.0.3
Copyright 2013 Yahoo!
Licensed under the BSD License.
https://github.com/pure-css/pure/blob/master/LICENSE.md
*/
```

Whereas it is after that PR `https://github.com/pure-css/pure/blob/master/LICENSE`.

**Comment by redonkulus** [comment]
Thanks!
